src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -217,6 +217,7 @@ public sealed partial class JsonSerializerOptions
         public System.Text.Json.JsonCommentHandling ReadCommentHandling { get { throw null; } set { } }
         public bool WriteIndented { get { throw null; } set { } }
         public System.Text.Json.Serialization.JsonConverter GetConverter(System.Type typeToConvert) { throw null; }
+        public System.Text.Encodings.Web.JavaScriptEncoder Encoder { get { throw null; } set { } }
```

Comment:
One of the errors of the PRs are that you are adding this to the ref but you aren't adding the implementation, so APICompat will fail.

Reply:
Yes, of course, this PR is WIP (the implementation wasn't done yet).